edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/deployment/KubernetesDeploymentMapper.cs

```diff
@@ -239,6 +253,10 @@ List<V1EnvVar> CollectModuleEnv(KubernetesModule module, IModuleIdentity identit
                 envList.Add(new V1EnvVar(KubernetesConstants.ProxyTrustBundlePathEnvKey, this.proxyTrustBundlePath));
                 envList.Add(new V1EnvVar(KubernetesConstants.ProxyTrustBundleVolumeEnvKey, this.proxyTrustBundleVolumeName));
                 envList.Add(new V1EnvVar(KubernetesConstants.ProxyTrustBundleConfigMapEnvKey, this.proxyTrustBundleConfigMapName));
+                envList.Add(new V1EnvVar(KubernetesConstants.ProxyConfigMemoryLimitKey, this.proxyMemoryLimit));
```

Comment:
I am not super clear how the environment variables are used.

Reply:
As we talked, let's use an k8s app. level setting to configure the limit. It'll be cleaner.

Reply:
Done - see the updated description of the PR.